src/index.android.tsx

```diff
@@ -98,13 +99,26 @@ const Navigator = DrawerNavigator({
     },
 });
 
+const RootNavigator = StackNavigator({
```

Comment:
why didn't you add this to the `DrawerNavigator` above with a null label?  I don't know a ton about reactnavigation so I don't know what the right thing to do is, just wondering.

Reply:
I saw in several issues that this is the recommended structure and the `DrawerNavigator` uses the keys to be shown as menu items:
```const Navigator = DrawerNavigator({
    Devices: {
        screen: DevicesNavigator,
    },
    Groups: {
        screen: GroupsNavigator,
    },
    Logout: {
        screen: LogoutScreen,
        navigationOptions: {
            drawerLabel: 'Logout',
        },
    },
    Login: {
        screen: LoginScreen,
        navigationOptions: {
            title: '',
            drawerLabel: '',
        },
    },
});```